src/conmon.c

```diff
@@ -728,6 +701,90 @@ static gboolean conn_sock_cb(int fd, GIOCondition condition, gpointer user_data)
 	return G_SOURCE_REMOVE;
 }
 
+static gboolean conn_sock_cb(G_GNUC_UNUSED int fd, GIOCondition condition, gpointer user_data);
+
+static void sock_try_write_to_masterfd_stdin(struct conn_sock_s *sock)
+{
+	if (!sock->remaining)
+		return;
+
+	ssize_t w = write(masterfd_stdin, sock->buf + sock->off, sock->remaining);
+	if (w < 0) {
+		nwarn("Failed to write to container stdin");
+	} else {
```

Comment:
stylistically, this is inconsistent with line 727. I don't mind which, but we should probably choose one for the project :smile: